### ISSUE TITLE
Finalise refund on termination clause

### ISSUE THREAD

**Issue by frabcus** [scraperwiki/pdftables-terms#2]
<blockquote>
"refund any outstanding credit within 30 days" could be quite a large liability years in the future, with quite a few people with unused credit.
</blockquote>

francisdavey added a note 4 days ago

<blockquote>
Yes, that's true. Worse than simply pro rata for a monthly subscription.

However, being able to take someone's money for a service and then not give them the service while keeping the money not only seems unattractive it might actually be unlawful in certain circumstances. Do you want to amend this to "we will not". I've softened it a little in draft.
</blockquote>

**Comment by pwaller** [comment]
Link to relevant fragment where discussion began (I struggled to find this a bit @frabcus): https://github.com/scraperwiki/pdftables-terms/pull/1#discussion_r33852507

**Comment by pwaller** [comment]
It's unclear to me exactly how to action this issue.

I guess there are two things. 1) is clause 31, which talks about termination for reasons beyond our control and what to do with refunds in that case. 2) in my mind, there is a problem about what to do about mass termination in general, which is separate from 31.

Currently [this reads](https://github.com/scraperwiki/pdftables-terms/blob/master/_terms_inner.html#L260-L270):

``` html
<p><span class="section-number">31</span> There are unfortunately
circumstances, which we think will be extremely unlikely to
occur, where we may have to terminate your access - for example
where we have been ordered to do so by a competent authority and
we are not permitted to tell you why. For this reason, we reserve
the right to terminate your access to the service at any time
without the need to give you a reason, although we will try to
give one if possible. In the unlikely event that we terminate
your access in this way where you are not at fault, we will
refund any outstanding credit within 30 days if we are permitted
to do so.</p>
```

I find the language at the end "if we are permitted to do so" a bit strange. Who could possibly not permit us to pay back? Our creditors, if there were any, I suppose?

That aside, let's pick apart 1 and 2.

1) Aside from what seems like strange language to me, the clause seems fine. I would be okay to take it as is.

2) Separately from clause 31, should/can we have some protection against large amounts of credit accruing? This protection could either be on a contractual basis via the T&Cs, or maybe we could think about enforcing it technically. The former seems more easier and more useful since it could potentially protect us years down the line if the site shut down and we were unable to make contact with everyone in a timely fashion to everyone who has credit stored with us.

Is there a way of specifying in the T&Cs that pages amount to a balance which is non-refundable, or only refundable in specific circumstances or at our choosing?

My concern is that over the course of years, we could have thousands of people who have partial balances of pages with us. If we announced that we had to change something and people were entitled to demand pro-rata refunds, that could cause a big mess.
